heka/files/lua/decoders/contrail_api_stdout_log.lua

```diff
@@ -0,0 +1,61 @@
+require "string"
+require "table"
+local l      = require 'lpeg'
+l.locale(l)
+
+local patt = require 'patterns'
+local contrail = require 'contrail_patterns'
+local utils  = require 'lma_utils'
+local table_utils = require 'table_utils'
+
+local default_severity = read_config('default_severity') or 'NOTICE'
+
+local msg = {
+    Timestamp   = nil,
+    Type        = 'log',
+    Hostname    = nil,
+    Payload     = nil,
+    Pid         = nil,
+    Fields      = nil,
+    Severity    = nil,
+}
+
+function process_message ()
+    local log = read_message("Payload")
+    local logger = read_message("Logger")
+    local service = string.gsub(logger, '.log', '')
+
+    local m
+    local m1
+    local m2
```

Comment:
iiuc you can use only the 'm' variable since the code doesn't use 'm' after setting 'm1' (and the same for 'm2').